test/unit/query-interface/delete.test.ts

```diff
@@ -39,4 +39,116 @@ describe('QueryInterface#delete', () => {
 
     expect(firstCall.args[1]?.bind).to.be.undefined;
   });
+
+  it('returns fields specified in `options.returning`', async () => {
```

Comment:
prefer moving these tests to `unit/query-generator/delete-query.test.ts` instead. QueryInterface only tests that replacements and bind parameters are handled correctly, but they are not relevant to your PR